lms/templates/calculator/toggle_calculator.html

```diff
@@ -0,0 +1,133 @@
+                      <td>${_("Integers")}<br />
+                        ${_("Fractions")}<br />
+                        ${_("Decimals")}
+                      </td>
+                      <td>2520<br />
+                        2/3<br />
+                        3.14, .98
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Operators")}</th>
+                      ## Translators: Please do not translate mathematical symbols.
+                      <td>${_("+ - * / (add, subtract, multiply, divide)")}<br />
+                        ## Translators: Please do not translate mathematical symbols.
+                        ${_("^ (raise to a power)")}<br />
+                        ## Translators: Please do not translate mathematical symbols.
+                        ${_("_ (add a subscript)")}<br />
+                        ## Translators: Please do not translate mathematical symbols.
+                        ${_("|| (parallel resistors)")}
+                      </td>
+                      <td>x+(2*y)/x-1
+                        x^(n+1)<br />
+                        v_IN+v_OUT<br />
+                        1||2
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Greek letters")}</th>
+                      <td>${_("Name of letter")}</td>
+                      <td>alpha<br />
+                        lambda
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Constants")}</th>
+                      <td>c, e, g, i, j, k, pi, q, T</td>
+                      <td>20*c<br />
+                        418*T
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Affixes")}</th>
+                      <td>${_("Percent sign (%) and metric affixes (d, c, m, u, n, p, k, M, G, T)")}</td>
+                      <td>20%<br />
+                        20c<br />
+                        418T
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Basic functions")}</th>
+                      <td>abs, exp, fact or factorial, ln, log2, log10, sqrt</td>
+                      <td>abs(x+y)<br />
+                        sqrt(x^2-y)
+                      </td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Trigonometric functions")}</th>
+                      <td>sin, cos, tan, sec, csc, cot<br />
+                        arcsin, sinh, arcsinh, etc.<br />
+                      </td>
+                      <td>sin(4x+y)<br />
+                        arccsch(4x+y)
+                      </td>
+                      <td></td>
+                    </tr>
+                    <tr>
+                      <th scope="row">${_("Scientific notation")}</th>
```

Comment:
Let's provide context for this: "Translators: Please see http://en.wikipedia.org/wiki/Scientific_notation"